resolwe/flow/views/entity.py

```diff
@@ -73,9 +73,10 @@ def order_queryset(self, queryset):
             fields = [field.strip() for field in ordering.split(",")]
             for match in filter(None, map(regex.match, fields)):
                 field_id = match.group("field_id")
+                # Always filter by user-visible attribute label.
                 annotation_value = AnnotationValue.objects.filter(
```

Comment:
You should rename this to `annotation_label`.

Reply:
Done.